node/node.go

```diff
@@ -152,8 +166,13 @@ func (n *Node) Run(ctx context.Context) {
 	client := feeder.NewClient(n.cfg.Network.URL())
 	synchronizer := sync.New(n.blockchain, adaptfeeder.New(client), n.log)
 	http := makeHttp(n.cfg.RpcPort, rpc.New(n.blockchain, n.cfg.Network), n.log)
+
 	n.services = []service.Service{synchronizer, http}
 
+	if n.cfg.Pprof {
+		n.services = append(n.services, pprof.New(defaultPprofPort, n.log))
```

Comment:
@omerfirmak @IronGauntlets is it expected way of running services like pprof here or should I move it somewhere else?